src/screens/MyTrainingsExcercisesScreen.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+import { View, SafeAreaView, ScrollView } from "react-native";
+import { Button, useTheme, Text, Card } from "react-native-paper";
+import { styles } from "../styles/globalStyles";
+import { useAppDispatch, useAppSelector } from "../app/store";
+import { loadTrainingPlans, selectTrainingPlansState } from "../features/trainingPlans";
+import { serverTimestamp } from "firebase/firestore";
+
+export const MyTrainingsExcercisesScreen = ({ navigation, route }) => {
+  const { planId } = route.params;
+  const theme = useTheme();
+  const dispatch = useAppDispatch();
+  const { trainingPlans } = useAppSelector(selectTrainingPlansState);
+  const [trainingPlan, setTrainingPlan] = useState(null);
+
+  useEffect(() => {
+    if (!trainingPlans?.length) {
+      dispatch(loadTrainingPlans())
+    } else {
+      const plan = trainingPlans.find(plan => plan.id === planId)?.data()
+      setTrainingPlan(plan)
+    }
+  }, [trainingPlans])
+
+  const convertTimestamp = (timestamp) => {
+    let date = timestamp.toDate();
+    let mm = date.getMonth();
+    let dd = date.getDate();
+    let yyyy = date.getFullYear();
+
+    date = `${dd}-${mm}-${yyyy}`;
```

Comment:
And just return this string. Also you can remove the variables above and call it just in the string literal.